rtlive/sources/data_it.py

```diff
@@ -1,39 +1,210 @@
+import io
 import logging
 import pandas
+import requests
+
+from datetime import datetime, timedelta
 
 from . import ourworldindata
 from .. import preprocessing
 
 _log = logging.getLogger(__file__)
 
+IT_REGION_NAMES = {
+    '1': 'Piemonte',
+    '2': 'Valle d\'Aosta',
+    '3': 'Lombardia',
+    '5': 'Veneto',
+    '6': 'Friuli Venezia Giulia',
+    '7': 'Liguria',
+    '8': 'Emilia-Romagna',
+    '9': 'Toscana',
+    '10': 'Umbria',
+    '11': 'Marche',
+    '12': 'Lazio',
+    '13': 'Abruzzo',
+    '14': 'Molise',
+    '15': 'Campania',
+    '16': 'Puglia',
+    '17': 'Basilicata',
+    '18': 'Calabria',
+    '19': 'Sicilia',
+    '20': 'Sardegna',
+    '21': 'P.A. Bolzano',
+    '22': 'P.A. Trento',
+    'all': 'Italy',
+}
+
+IT_REGION_CODES = {
+    v : k
+    for k, v in IT_REGION_NAMES.items()
+}
+
+IT_REGION_POPULATION = {
+    'all': 60_244_639,
+    '1': 4_341_375,
+    '2': 125_501,
+    '3': 10_103_969,
+    '5': 4_907_704,
+    '6': 1_211_357,
+    '7': 1_543_127,
+    '8': 4_467_118,
+    '9': 3_722_729,
+    '10': 880_285,
+    '11': 1_518_400,
+    '12': 5_865_544,
+    '13': 1_305_770,
+    '14': 302_265,
+    '15': 5_785_861,
+    '16': 4_008_296,
+    '17': 556_934,
+    '18': 1_924_701,
+    '19': 4_968_410,
+    '20': 1_630_474,
+    '21': 532_080,
+    '22': 542_739,
+}
+
+LABEL_TRANSLATIONS = {
+    "curves_ylabel": "Giornalieri",
+    "testcounts_ylabel": "Tests",
+    "probability_ylabel": "Probabilità\n$R_t$>1",
+    "rt_ylabel": "$R_t$",
+    "curve_infections": "Infezioni",
+    "curve_adjusted": "Test positivi attesi",
+    "bar_positive": "Test positivi confermati",
+    "bar_actual_tests": "Dati",
+    "curve_predicted_tests": "Previsione",
+}
```

Comment:
If I'm not mistaken, this variable is not used anywhere in this file

Reply:
This is probably copied from `data_de`. We use the `LABEL_TRANSLATIONS` to make a localized version of the plots for the German dashboard. It's not part of the global API, but it doesn't hurt to keep this.

Reply:
> Hi @davideferre, thanks for this contribution!
> 
> I added one comment.
> Can you post a screenshot of the plot from the dataloading notebook?
> 
> Looks great!

Hi @michaelosthege I'm not sure if I have understood correctly what you ask me. Do you need someting generated after running this model with italian data?

Reply:
> This is probably copied from `data_de`. We use the `LABEL_TRANSLATIONS` to make a localized version of the plots for the German dashboard. It's not part of the global API, but it doesn't hurt to keep this.

Exactly, I've add this dictionary if needed in some front-end strings

Reply:
> This is probably copied from data_de. We use the LABEL_TRANSLATIONS to make a localized version of the plots for the German dashboard. It's not part of the global API, but it doesn't hurt to keep this.

Ah good point. I should probably add this for France actually 🤔

Reply:
> Hi @michaelosthege I'm not sure if I have understood correctly what you ask me. Do you need someting generated after running this model with italian data?

Michael is just asking that you run [this NB](https://github.com/rtcovidlive/rtlive-global/blob/master/notebooks/Tutorial_dataloading.ipynb) with the new Italian data and post a screenshot of the plots you get, to make sure it works I guess

Reply:
I just did that and it looks good ✔️